<?xml version="1.0"?>

<launch>
  <arg name="joy_dev" default="/dev/input/js0" />
  <arg name="max_linear_velocity" default="0.3" />
  <arg name="max_angular_velocity" default="1.5" />

  <node pkg="joy" type="joy_node" name="joy_node" output="screen">
    <param name="dev" value="$(arg joy_dev)" />
  </node>

  <node pkg="nav_lecture" type="teleop_joy.py" name="teleop_joy" output="screen" >
    <param name="max_linear_velocity" value="$(arg max_linear_velocity)" />
    <param name="max_angular_velocity" value="$(arg max_angular_velocity)" />
  </node>
</launch>
